handbook/people-ops/performance-reviews.md

```diff
@@ -1,3 +1,56 @@
 # Performance reviews
 
-TODO(@sourcegraph/people-ops): document this
+## Why we do review cycles
+The goal of these review cycles is to ensure that we’re providing every teammate the resources they need to continue to grow during their time at Sourcegraph, and review cycles offer a unique opportunity to give and receive valuable feedback from managers, direct reports, and peers. By having biannual 360 reviews, we aim to get everyone the feedback necessary to help them progress in their career and meet their professional goals. 
```

Comment:
get -> give
```suggestion
The goal of these review cycles is to ensure that we’re providing every teammate the resources they need to continue to grow during their time at Sourcegraph, and review cycles offer a unique opportunity to give and receive valuable feedback from managers, direct reports, and peers. By having biannual 360 reviews, we aim to give everyone the feedback necessary to help them progress in their career and meet their professional goals. 
```

I think we should clarify that we expect managers to continuously deliver feedback and help with professional development (e.g. in weekly 1-1s), and lattice review cycles are just another part of this continuous process.